docs/rest_api.md

```diff
@@ -173,6 +169,92 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, indexed by the name of the network with interior dictionaries indexed by project and channel.
```

Comment:
1. Please hard-wrap this line to 72-80 chars.
2. What does it mean to be indexed by both project and channel? Keys are just strings, so it's not clear what this means.
3. Your example below does not agree with this description.